leftwm-core/src/handlers/window_handler.rs

```diff
@@ -378,22 +367,12 @@ fn setup_window(
             }
         }
 
-        // Setup a child window.
-        if let Some(parent) = find_transient_parent(&state.windows, window.transient) {
-            // This is currently for vlc, this probably will need to be more general if another
-            // case comes up where we don't want to move the window.
-            if window.r#type != WindowType::Utility {
-                set_relative_floating(window, ws, parent.exact_xyhw());
-                return;
-            }
-        }
-
```

Comment:
I tried to find the use-case with `vlc` for that but even after removing it all dialogues were fine with `vlc`. Could anyone elaborate which case this if-clause catches?

Reply:
The play pause popup that appears when vlc is in fullscreen. We obviously dont want to centre it

Reply:
Tested vlc specifically and it seems to work correct at first, but when you click any empty space of the play-controls window during fullscreen, it jumps to a position more or less centered.

Reply:
Also to add to this, the parent clause is for any application and centres the dialog on the window

Reply:
Yeah, in the case where there is a parent we want to centre on the parent. However if when centring, we go outside the workspace (like the edge of the window being off the screen, what `xyhw.contains_xyhw()` checks), we would want to centre on the workspace. Here is an example with android studio where the dialog is centred on the app, which is nicer:
![example](https://user-images.githubusercontent.com/63980259/192755116-9c6567ea-ab15-4f72-a851-575be67bae81.png)

Reply:
Hm.... ok I understand what you mean now. I'll fix that